dev/docker/web/epidata/Dockerfile

```diff
@@ -7,3 +7,6 @@ COPY repos/delphi/delphi-epidata/src/server/*.php /var/www/html/epidata/
 
 # point to the development database (overwrites the production config)
 COPY repos/delphi/delphi-epidata/dev/docker/web/epidata/assets/database_config.php /var/www/html/epidata/
+
+RUN chmod o+r /var/www/html/secrets.php
```

Comment:
the `secrets.php` file is added by a separate dockerfile here: https://github.com/cmu-delphi/operations/blob/main/dev/docker/web/Dockerfile

i recommend updating permissions there instead of here, because there are other docker images besides this one that are built on top of that.

Reply:
If we update the permissions in operations, will that propagate here?